src/AWS.Deploy.CLI/Commands/CommandHandlerInput/GenerateDeploymentProjectCommandHandlerInput.cs

```diff
@@ -0,0 +1,16 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AWS.Deploy.CLI.Commands.CommandHandlerInput
+{
+    /// <summary>
+    /// This class maps the command line options for the "deployment-project generate" command to the appropriate C# properties.
+    /// </summary>
+    public class GenerateDeploymentProjectCommandHandlerInput
+    {
+        public string? ProjectPath { get; set; }
+        public bool Diagnostics { get; set; }
+        public string? Output { get; set; }
+        public string? ProjectDisplayName { get; set; }
```

Comment:
why do we need display name as param?

Reply:
I suggest we need a display name so when we save the CDK project we having something to put in the Name field of the recipe file. I want to avoid after saving a deployment project that we don't have 2 recommendations that look the same when we go back and deploy.

Reply:
I have included a CLI argument called `--project-display-name` which the customers can use to give a name to their deployment project. This name will be displayed when listing out different deployment recommendations